api/rest/v2/getting_started.md

```diff
@@ -1,273 +1,10 @@
 ---
-layout: default
+layout: redirect
 title: Getting Started Guide
 nav_order: 0
 parent: REST (v2)
 grand_parent: API
-published: true
+published: false
```

Comment:
this needs to be true for the redirect to work

Reply:
What if I just remove this altogether?